src/index/entry.rs

```diff
@@ -27,14 +27,15 @@ pub(crate) struct InscriptionEntry {
   pub(crate) number: u64,
   pub(crate) sat: Option<Sat>,
   pub(crate) timestamp: u32,
+  pub(crate) inscription_fee: u64,
```

Comment:
How about `fee` instead of `inscription_fee`? Also, these should be sorted.

```suggestion
  pub(crate) fee: u64,
```